www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -62,19 +65,22 @@ function initDatepicker(className, altFormat, defaultDate, idName, timestampToSe
             var altName = jQuery(this).attr("name");
             if (typeof(altName) != "undefined") {
                 var alternativeField = "input[name=alternativeDate" + altName[0].toUpperCase() + altName.slice(1) + "]";
-                var value = $(this) && $(this).val() ? $(this).val() : defaultDate;
+                // $(this).val(), if exists, is a GMT YYYY-MM-DDTHH:mm:ss timestamp, for example with PHP : gmdate("Y-m-d\TH:i:s")
+                var value = $(this) && $(this).val() ? moment($(this).val(), "DD-MM-YYYY").tz(timezone) : defaultDate;
```

Comment:
IMO `, "DD-MM-YYYY"` you added is incorrect / not needed.
Look at the comment above (and also line 40) saying that a GMT `YYYY-MM-DDTHH:mm:ss` timestamp is expected.

Reply:
Hi @UrBnW ,
The format was added to resolve the deprecated moment notice when submitting the form :
![image](https://user-images.githubusercontent.com/34628915/77639889-bce33200-6f59-11ea-94eb-3f22ff3b9268.png)

Reply:
The modification is related to this issue : https://github.com/moment/moment/issues/1407

Reply:
Do you know where this value was provided from ?
I'm pretty sure not from downtimes, as downtime dates are not computed by the backend anymore, but by the frontend / Moment.js.
Perhaps we should locate the culprit(s) and modify the format they use to feed the datepicker, to a proper GMT `YYYY-MM-DDTHH:mm:ss` one ?

Reply:
You should then need to merge https://github.com/centreon/centreon/pull/8289 before :)

Reply:
it's already merger on the master. specific PR are required to add theses modifications on the other branches